client/src/app/core/ui-services/projection-dialog.service.ts

```diff
@@ -55,10 +55,12 @@ export class ProjectionDialogService {
             const [action, projectors, projectorElement]: ProjectionDialogReturnType = response;
             if (action === 'project') {
                 this.projectorService.projectOnMultiple(projectors, projectorElement);
+                return { fullscreen: projectorElement.fullscreen, displayType: projectorElement.displayType };
```

Comment:
This is countdown related code!
Please make this more generic and extract all extra properties from the projector Element.